### DISCUSSION TITLE
NEED IIS HELPS TO DEPLOY/LAUNCH ANY DEMO ADONIS PROJECT ON IIS WITH IISNODE PLS

### DISCUSSION THREAD

**Discussion by MarkBerezowski** [adonisjs/core#4306] [Help]
We have a problem hosting an Adonis project on IIS server.  error message : Exception: E_INVALID_ENV_VALUE: Value for environment variable "PORT" must be numeric, instead received "\\.\pipe\2dccb6be-55ag-4y44-bc1a-1e62a1w0d9ca". We need to make it work with pipes instead of port. we have tried many web.config but nothing helps.  Any help

**Comment by McSneaky** [comment]
You can put it to host instead. You can also edit env validation in your `env.ts` file and change it to your liking
